airflow/models/__init__.py

```diff
@@ -2812,16 +2821,39 @@ def get_default_view(self):
             return self.default_view
 
     @provide_session
-    def get_last_dagrun(self, session=None, include_externally_triggered=False):
-        return get_last_dagrun(self.dag_id, session=session,
-                               include_externally_triggered=include_externally_triggered)
+    def get_last_dagrun(self, include_externally_triggered=False, session=None):
```

Comment:
why are we switching the kwargs order? Base on provide_session, it shouldn't matter?

Reply:
Well only did this because In most methods session is in the end, just for consistency.